.kuzzlerc

```diff
@@ -51,12 +51,50 @@
       "_id": "admin",
       "roles": [ "admin" ]
     },
+    "default": {
+      "_id": "default",
+      "roles": [ "default" ]
+    },
     "anonymous": {
       "_id": "anonymous",
-      "roles": [ "guest" ]
+      "roles": [ "anonymous" ]
     }
   },
-  // Default security roles
+  // role template for all users before to create the first admin
+  "roleWithoutAdmin": { 
+    "indexes": {
```

Comment:
missing "_canDelete" policy under indexes.* and indexes.collections.*  (see sample from "admin" role)